server/src/datasources/postgresdb.datasource.ts

```diff
@@ -0,0 +1,35 @@
+import {inject, lifeCycleObserver, LifeCycleObserver} from '@loopback/core';
+import {juggler} from '@loopback/repository';
+import * as dotenv from 'dotenv';
+dotenv.config();
+
+const config = {
+  name: 'postgresdb',
+  connector: 'postgresql',
+  // url: 'postgresql://$vipul:$vipul123@$localhost:$5432/$myLoopbackdb',
+  host: process.env.HOST,
+  port: process.env.PORT,
+  user: process.env.DBUSERNAME,
+  password: process.env.PASSWORD,
```

Comment:
make a .env.example file for info of all relevant values

Reply:
added .env.example file